config/cops.yml

```diff
@@ -1,24 +1,25 @@
 Metrics/AbcSize:
   violation_points: 100_000
 Metrics/BlockNesting:
-  base_points: 30_000
+  base_points: 100_000
 Metrics/ClassLength:
-  base_points: 400_000
+  base_points: 800_000
+  violation_points: 75_000
 Metrics/CyclomaticComplexity:
-  base_points: 75_000
-  violation_points: 10_000
+  base_points: 1_000_000
```

Comment:
2,000,000 remediation points are needed in penalties to go from A to B. 

So not always, but a lot of the time an instance of Cyclomatic Complexity could have that large of grade impact.

I was talking with @brynary yesterday and had the impression that that level of grade impact would be _close_ to what we want for complexity - but maybe requires some additional discussion?